src/main/java/org/eclipsefoundation/marketplace/service/impl/GuavaCachingService.java

```diff
@@ -47,43 +48,62 @@
 public class GuavaCachingService<T> implements CachingService<T> {
 	private static final Logger LOGGER = LoggerFactory.getLogger(GuavaCachingService.class);
 
-	@ConfigProperty(name = "cache.max", defaultValue = "2500")
+	@ConfigProperty(name = "cache.max", defaultValue = "10000")
```

Comment:
I would name it more explicitly `cache.max.size`, because "max" could be for many cache parameters.